functions.opengraph.php

```diff
@@ -94,6 +94,24 @@ function jetpack_og_tags() {
 			$tags['profile:first_name'] = get_the_author_meta( 'first_name', $author->ID );
 			$tags['profile:last_name']  = get_the_author_meta( 'last_name', $author->ID );
 		}
+	} elseif ( is_archive() ) {
+		$tags['og:type'] 		= 'website';
```

Comment:
I think we should respect what can be saved in options, like we do in other places where the type is set:

```suggestion
		$tags['og:type']         = ! empty( $site_type ) ? $site_type : 'website';
```

Reply:
I agree that deferring to the saved option makes sense for the home page. However, it seems that an archive page doesn't fit neatly into any of the standard OG options (https://ogp.me/#types). Both the OG protocol and Facebook (https://developers.facebook.com/docs/sharing/webmasters#markup) suggest a default of "website".

Does Jetpack have namespaced types that it's using instead of the standard?  I'm trying to find what the options for open_graph_protocol_site_type are, but haven't been able to locate them.

Reply:
Yeah, you're right. Let's keep it with that default. 👍